example/android/app/build.gradle

```diff
@@ -26,16 +26,19 @@ apply plugin: 'kotlin-android'
 apply from: "$flutterRoot/packages/flutter_tools/gradle/flutter.gradle"
 
 android {
+
     compileSdkVersion flutter.compileSdkVersion
-    ndkVersion flutter.ndkVersion
+    ndkVersion = "25.1.8937393"
```

Comment:
Shouldn't we always use what flutter provides?

Reply:
Arcgis requires a higher ndkVersion, so I just bumped that. 👍🏽